src/index.tsx

```diff
@@ -3,6 +3,7 @@ import { Provider } from 'mobx-react';
 import { createRoot } from 'react-dom/client';
 import { BrowserRouter } from 'react-router-dom';
 import App from '~app/App';
+import GTMFrame from '~root/GTMFrame';
```

Comment:
Root is not a good place to put this tsx file, and any other tsx except App.
Move it to `src/lib/analytics/GoogleTag/components/GTMFrame`
Also there's existing `GoogleTagManager` class in analytics folder, it should be also moved to `src/lib/analytics/GoogleTag/GoogleTagManager`

Reply:
done